module/x/gravity/types/ethereum.go

```diff
@@ -63,7 +63,7 @@ func NewSDKIntERC20Token(amount sdk.Int, contract common.Address) ERC20Token {
 
 // GravityCoin returns the gravity representation of the ERC20
 func (e ERC20Token) GravityCoin() sdk.Coin {
-	return sdk.Coin{Amount: e.Amount, Denom: strings.Join([]string{GravityDenomPrefix, e.Contract}, GravityDenomSeparator)}
+	return sdk.Coin{Amount: e.Amount, Denom: strings.Join([]string{GravityDenomPrefix, strings.ToLower(e.Contract)}, GravityDenomSeparator)}
```

Comment:
This seems like a generally good idea and should help with weird issues around capitalization of eth addresses.

Reply:
Is there any concern that fields from `GravityCoin` end up in state anywhere that could cause a divergence based on the capitalization?

Reply:
> Is there any concern that fields from `GravityCoin` end up in state anywhere that could cause a divergence based on the capitalization?

the unit test I added fails without this change, the `Denom` field is used to create coins in the bank module.